### ISSUE TITLE
UnitTest課題が、結合テストと勘違いしそう

### ISSUE THREAD

**Issue by novr** [yumemi-inc/ios-training#63]
https://github.com/yumemi-inc/ios-training/blob/main/Documentation/UnitTest.md について

`テストコードを書くためにリファクタリングする部分` の例が
Modelについて書かれているためViewのテストをすべきところ、View+Modelの結合テストを想定してしまう
https://github.com/yumemi-inc/ios-training-bjung/pull/8#discussion_r1451867708

ViewのUnitTestであることを明記すべきか？

**Comment by el-hoshino** [comment]
確かに、Testって言われてもさまざまなテストがあるから非常に曖昧ですので、UnitTestって明記した方がいいと思いました 🙋

**Comment by macneko-ayu** [comment]
これ、どういうコードを書いたら正解なのかを私も理解できていなくて、[OJT](https://yumemi.slack.com/archives/C02HU3K4RU1/p1706690476671469)で悩んでたんですが、ここで聞いちゃうのが早そう。
「スタブなりで用意したModelをVCに渡して、UIImageViewに晴れ画像が表示されること」がゴールであって、「ロジックを通して生成されたModelをVCに渡して、UIImageViewに晴れ画像が表示されること」は求めていない、という認識であってますか…？

**Comment by novr** [comment]
`画面に晴れ画像が表示されること` を確認しようとするとUITestになってしまうので
文言も揃えてこんな感じかな

- 天気予報がxxxだったら、画面にxxx画像が設定されること
- 天気予報の最高気温が画面に設定されること
